src/app/_hooks/useGetPosters.ts

```
@@ -0,0 +1,62 @@
+  })
+
+  const [languageFilter, favoriteFilter] = [searchParams.get('language'), searchParams.get('favorite')]
+
+  const postersData = useMemo(() => {
+    const allPosters: PosterWithFavorite[] | undefined = data?.movies.search.edges?.[0]?.node?.images.posters
+
+    if (!languageFilter && !favoriteFilter) {
+      return allPosters
+    }
+
+    if (!!languageFilter) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)